modules/nw-anp-np-reference.adoc

```diff
@@ -0,0 +1,65 @@
+// Module included in the following assemblies:
+//
+// * networking/openshift_network_security/logging-network-security.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="nw-anp-differences-networkpolicy_{context}"]
+= Key differences between AdminNetworkPolicy and NetworkPolicy custom resources
+
+[cols="1,1,1"]
+|===
+|Policy elements | AdminNetworkPolicy | NetworkPolicy
+
+|Applicable user
+|Cluster administrator or equivalent
+|Developers within namespaces
+
+|Scope
+|Cluster
+|Namespaced
+
+|Drop traffic
+|Supported with an `Deny` action set as a rule.
+|Supported via implicit isolation of target pods.
+
+|Skip enforcement
+|Supported with an `Pass` action set as a rule.
+|Not applicable
+
+|Allow traffic
+|Supported with an `Allow` action set as a rule.
+|The default action for all rules is to allow.
+
+|Implicit isolation
+|No implicit isolation
+|All rules have an implicit isolation of target pods.
+
+|Rule precedence
+|Depends on the order in which they appear within an ANP.
+|Rules are additive
+
+|Policy precedence
+|Among ANPs the priority field sets the order for evaluation. As a tier 1 ACL policy, ANPs are enforced before any `NetworkPolicy` objects.
+|They are enforced after ANP and before BANP. The are evaluated in tier 2 of the ACL.
+
```

Comment:
Insert another row as "Feature precedence" and add under ANP column, evaluated first via Tier1ACLs (BANP is evaluated last via Tier3 ACLs) and for netpol you can say "They are enforced after ANP and before BANP. The are evaluated in tier 2 of the ACL."